tools/test-proxy/Azure.Sdk.Tools.TestProxy/Admin.cs

```diff
@@ -64,7 +65,7 @@ public async void AddTransform()
         }
 
         [HttpPost]
-        public async void AddSanitizer()
+        public async Task AddSanitizer()
```

Comment:
This fix seems to fix an issue I was running into while testing.

Reply:
merging this today after commiting your feedback and doing a quick sanity test.